Windows error in integration tests

https://github.com/nodejs/node/blob/842e5a10ac59c826f875ef79e92d144e2655cc9a/test/common.js#L502-L505

```
 hmr
    hmr server
error Command failed with exit code 3221225477.
      √ should emit an HMR update for the file that changed (423ms)
      √ should emit an HMR update for all new dependencies along with the changed file (219ms)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 3221225477.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 3221225477
Command: C:\hostedtoolcache\windows\node\14.21.3\x64\node.exe
Arguments: C:\npm\prefix\node_modules\yarn\lib\cli.js test-ci
Directory: D:\a\parcel\parcel\packages\core\integration-tests
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
error Command failed with exit code 3221225477.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/packages/core/cache/package.json
+++ b/packages/core/cache/package.json
@@ -27,7 +27,7 @@
     "@parcel/fs": "2.8.3",
     "@parcel/logger": "2.8.3",
     "@parcel/utils": "2.8.3",
-    "lmdb": "2.5.2"
+    "lmdb": "2.7.11"
   },
   "peerDependencies": {
     "@parcel/core": "^2.8.3"

